server/handlers/client.js

```diff
@@ -60,12 +61,26 @@ exports.addFundToWalletHandler = async function (req, res, next) {
   return res.json({ newWalletValue: result.value.wallet });
 };
 
-exports.findClientValidatorChain = [searchStringValidator];
+exports.findClientValidatorChain = [
+  searchStringValidator,
+  hasPendingCancelationValidator,
+];
 exports.findClientsHandler = async function (req, res, next) {
   let result;
 
+  let hasPendingCancelation = req.query.hasPendingCancelation;
+
+  if (hasPendingCancelation === "true") {
+    hasPendingCancelation = true;
+  } else if (hasPendingCancelation === "false") {
+    hasPendingCancelation = false;
+  }
```

Comment:
If req.query.hasPendingCancelation is not "true" or "false" but a different string what will happen?